packages/performance/src/services/remote_config_service.ts

```diff
@@ -170,15 +162,13 @@ function getRemoteConfig(
  * is valid.
  */
 function processConfig(
-  iid: string,
   config: RemoteConfigResponse | undefined
```

Comment:
can be written as 
function processConfig(config?: RemoteConfigResponse)

Reply:
Done.